charts/karpenter/values.yaml

```diff
@@ -185,8 +185,11 @@ settings:
   # Interruption handling is disabled if not specified. Enabling interruption handling may
   # require additional permissions on the controller service account. Additional permissions are outlined in the docs.
   interruptionQueue: ""
-  # -- Reserved ENIs are not included in the calculations for max-pods or kube-reserved
-  # This is most often used in the VPC CNI custom networking setup https://docs.aws.amazon.com/eks/latest/userguide/cni-custom-network.html
+  # -- Number of reserved ENIs to exclude in the calculations for max-pods or kube-reserved. This is most often used in the VPC CNI
+  # custom networking setup https://docs.aws.amazon.com/eks/latest/userguide/cni-custom-network.html and Security groups for Pods 
+  # https://docs.aws.amazon.com/eks/latest/userguide/security-groups-for-pods.html. Increase by one for each feature enabled. 
```

Comment:
Is it accurate to say that we should increase this value by 1 for each feature that is enabled since custom networking could have any number of reserved ENIs for the node host network

Reply:
Hey. The reason for setting reserved ENI is that primary ENI will not be used for pods IPs, so you lose one ENI worth of pod IPs. Likewise with trunk ENI you also lose one, due to VPC CNI no longer (1.16+) allocate pod IPs from trunk interface.

Reply:
Could put such additional explanation in docs, if not too long winded.